### PR TITLE
add isDisplayed method

### PR DESCRIPTION
**Pull request by NiluparErkin**

### CODE REVIEW

**Review by WenjiZ — DISMISSED** [review]
LGTM!

**Review by Jack-Vu — DISMISSED** [review]
LGTM!

**Review by Wei1150 — DISMISSED** [review]
LGTM!

File: src/test/java/utilities/Util.java [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+import org.openqa.selenium.WebElement;
+
 public class Util {
+	/*
+	 * Checks if an element is displayed
+	 * 
+	 * @Author: Nilupar Erkin
+	 */
+	
+	public static boolean isElementDisplayed(WebDriver driver, String xPathOfElement) {
```
Comment (boratech-macmini) [inline src/test/java/utilities/Util.java]: It would be a better idea to use By object instead of xpath, would be more reusable

**Review by boratech-macmini — CHANGES_REQUESTED** [review]

**Review by Wei1150 — APPROVED** [review]

**Review by boratech-macmini — APPROVED** [review]

**Pull request merged**